db/migrate/20240126213348_set_existing_form_state.rb

```diff
@@ -1,23 +0,0 @@
-class SetExistingFormState < ActiveRecord::Migration[7.1]
```

Comment:
If we add in 

```ruby
  class Form < ActiveRecord::Base
    self.table_name = :forms
  end
```

then this no longer fails

Reply:
The migration does nothing anyway, and a while ago we wanted to get rid of the `has_live_version` method anyway but I wasn't sure what to do about this migration - so I reckon let's delete anyway.